src/OmniSharp.Abstractions/OmniSharpEndpoints.cs

```diff
@@ -77,6 +77,8 @@ public static class V2
             public const string Highlight = "/v2/highlight";
 
             public const string GotoDefinition = "/v2/gotodefinition";
+
+            public const string GotoTypeDefinition = "/v2/gototypedefinition";
```

Comment:
@filipw Since this is the first iteration of this endpoint, is it ok to be under V2?

Reply:
I think it should just be `/gototypedefinition`, we do not version at API level but on endpoint level.
for example `/completion` is the modern version of intellisense and it was not versioned as `v2`

Reply:
Okay, it's updated accordingly